devlab/provision/generate_config.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+cloudferry_dir=$(pwd)
+cloudferry_dir=${cloudferry_dir%/CloudFerry*}/CloudFerry
```

Comment:
According to [abs guide](http://tldp.org/LDP/abs/html/string-manipulation.html) it deletes longest match of $substring from back of $string. It returns path to CloudFerry directory based on current dir... but it will actually fail if run from any other directory :( Let me fix it.